### PR TITLE
Add axes command to show cell axis directions

### LINKED ISSUE #9: [Feature request] Arrows for cell axes
Your `supercell` function is excellent. I'm using it in a course this semester! However, it is hard to tell which edge corresponds to which unit cell axis. Would it possible to add arrows corresponding to the cell axes. I'm thinking of something like [these scripts](https://pymolwiki.org/index.php/Axes) but with the cell axes as the basis rather than a cartesian basis. It would be great if it could be integrated into `xtal.py`.

### PR DESCRIPTION
**Pull request by kmdalton**
This pr resolves #9 by adding an axes command that shows the directions of the cell axes A, B, C in red, green, and blue arrows.

### CODE REVIEW

**Comment by kmdalton** [comment]
Hi, @speleo3, here is a very rudimentary function which draws VMD style arrows pointing in the cell axis directions. Certainly there could be some improvements. Supporting colorblind friendly arrows would be a good start. I figured it was better to get basic functionality first. Let me know what you think!

**Comment by speleo3** [comment]
Thanks @kmdalton, this looks good! We may want to keep the `axes` command compatible to the original https://pymolwiki.org/index.php/Axes code, e.g. don't grab `cmd.get_object_list()[0]` if object is `None`. How about adding a new command `axes_object` (or a similar name) for the new behavior?

**Comment by kmdalton** [comment]
@speleo3 -- I renamed the function to `cell_axes`. Does that address your compatibility concern?

I also added optional arguments for axis colors, and I changed the default colors to a [colorblind-friendly palette](https://colorbrewer2.org/#type=qualitative&scheme=Dark2&n=3). 

Looks pretty slick:
image

**Comment by kmdalton** [comment]
@speleo3 -- I cleaned this pr up a bit more, added tab-complete, and fixed a bug where the B-axis cone wasn't rendering for orthorhombic space groups. See [here](https://github.com/schrodinger/pymol-open-source/issues/220) for details, but I think there is a bug in `cgo`.

File: psico/xtal.py [hunk 1]
```diff
@@ -316,12 +316,129 @@ def biomolecule(name=None, filename=None, prefix=None, number=1, suffix=None,
...
 cmd.extend('supercell', supercell)
 cmd.extend('symexpcell', symexpcell)
 cmd.extend('biomolecule', biomolecule)
 
 # tab-completion of arguments
 cmd.auto_arg[0]['biomolecule'] = cmd.auto_arg[0]['pseudoatom']
 cmd.auto_arg[3]['supercell'] = cmd.auto_arg[0]['pseudoatom']
+cmd.auto_arg[0]["cell_axes"] = [cmd.object_sc, "object", ""]
 
-# vi: ts=4:sw=4:smarttab:expandtab
```
Comment (speleo3) [inline psico/xtal.py:327]: can you please keep this line?

File: psico/xtal.py [hunk 2]
```diff
@@ -316,12 +316,129 @@ def biomolecule(name=None, filename=None, prefix=None, number=1, suffix=None,
     cmd.disable(name)
     cmd.group('%s_%s' % (prefix, suffix), '%s_%s_*' % (prefix, suffix))
 
+from chempy import cpv
```
Comment (speleo3) [inline psico/xtal.py:319]: please move this import to the top of the file (after `from pymol import ...`)

File: psico/xtal.py [hunk 3]
```diff
@@ -316,12 +316,129 @@ def biomolecule(name=None, filename=None, prefix=None, number=1, suffix=None,
...
+        m = [z, 0, 0, 0, 0, z, 0, 0, 0, 0, z, 0, t[0] / z, t[1] / z, t[2] / z, 1]
+        cmd.set_object_ttt(self.name, m)
+
+def cell_axes(object=None, a_color='0xd95f02', b_color='0x1b9e77', c_color='0x7570b3', name='cell_axes'):
+    '''
+DESCRIPTION
+
+    Draw arrows corresponding to the crystallographic axes.
+
+    The default color palette is colorblind friendly but close to the familiar red, green, and blue for the a, b, and, c axes respectively.
```
Comment (speleo3) [inline psico/xtal.py:366]: I suggest to wrap this line (after "familiar").

**Review by speleo3 — CHANGES_REQUESTED** [review]
Very nice, I like it! `cell_axes` is a good name.

One question: Do you mind if I merge this with the "squash" option? That would combine your 7 commits into one, which produces a nicer git history and works better with git blame (for my taste). For the future, I also recommend to make a named branch in your fork, don't make pull requests from your "master" branch.

**Comment by kmdalton** [comment]
I don't mind if you squash the merge. I can make a branch next time. I believe I resolved the comments in your code review. Let me know if you need anything else from me. 

Cheers

**Review by speleo3 — APPROVED** [review]

**Pull request merged**